yt_dlp/extractor/prankcastpost.py

```diff
@@ -0,0 +1,70 @@
+from .common import InfoExtractor
+from ..utils import parse_iso8601, traverse_obj, try_call
+
+
+class PrankCastPostIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?prankcast\.com/[^/?#]+/posts/(?P<id>\d+)-(?P<display_id>[^/?#]+)'
+    _TESTS = [{
+        'url': 'https://prankcast.com/devonanustart/posts/6214-happy-national-rachel-day-',
+        'info_dict': {
+            'id': '6214',
+            'ext': 'mp3',
+            'title': 'Happy National Rachel Day!',
+            'display_id': 'happy-national-rachel-day-',
+            'timestamp': 1704333938,
+            'uploader': 'Devonanustart',
+            'channel_id': 4,
+            'duration': 13175,
+            'cast': ['Devonanustart'],
+            'description': '',
+            'categories': ['prank call'],
+            'tags': [''],
+            'upload_date': '20240104'
+        }
+    }, {
+        'url': 'https://prankcast.com/despicabledogs/posts/6217-jake-the-work-crow-',
+        'info_dict': {
+            'id': '6217',
+            'ext': 'mp3',
+            'title': 'Jake the Work Crow!',
+            'display_id': 'jake-the-work-crow-',
+            'timestamp': 1704346592,
+            'uploader': 'despicabledogs',
+            'channel_id': 957,
+            'duration': 263,
+            'cast': ['despicabledogs'],
+            'description': 'https://imgur.com/a/vtxLvKU',
+            'categories': [],
+            'tags': [''],
+            'upload_date': '20240104'
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id, display_id = self._match_valid_url(url).group('id', 'display_id')
+
+        webpage = self._download_webpage(url, video_id)
+        json_info = self._search_nextjs_data(webpage, video_id)['props']['pageProps']['ssr_data_posts']
+        json_post_info = self._parse_json(json_info['post_contents_json'], video_id)[0]
```

Comment:
bad var names imo. Maybe:
```suggestion
        post = self._search_nextjs_data(webpage, video_id)['props']['pageProps']['ssr_data_posts']
        content = self._parse_json(json_info['post_contents_json'], video_id)[0]
```